src/pages/admin/match.tsx

```diff
@@ -0,0 +1,160 @@
+import { eq } from "drizzle-orm";
+import { Elysia, t } from "elysia";
+import { type Session } from "lucia";
+import { HeaderHtml } from "../../components/header";
+import { LayoutHtml } from "../../components/Layout";
+import { NavbarHtml } from "../../components/Navbar";
+import { ctx } from "../../context";
+import {
+  deleteMatch,
+  getMatch,
+  getMatches,
+} from "../../db/queries/matchQueries";
+import { matches } from "../../db/schema";
+import { allTimeSeason } from "../../db/schema/season";
+import { isHxRequest, redirect } from "../../lib";
+import { fromTimezoneToUTC } from "../../lib/dateUtils";
+import { EditMatchModal } from "./components/EditMatchModal";
+import { MatchCard } from "./components/MatchCard";
+
+export const Match = new Elysia({
+  prefix: "/match",
+})
+  .use(ctx)
+  .get("/", async ({ html, session, headers }) => {
+    return html(() => matchPage(session, headers));
+  })
+  .get("/:id", async ({ params: { id }, session }) => {
+    const matchToEdit = await getMatch(Number(id), !!session?.user);
+    if (!matchToEdit) return;
+    return <EditMatchModal match={matchToEdit} />;
+  })
+  .put(
+    "/",
+    async ({ set, headers, body, writeDb }) => {
+      const createdAtFromUser = new Date(
+        `${body.date_played}T${body.time_played}`,
+      );
+      const createdAt = fromTimezoneToUTC(
```

Comment:
Lokalt virker det her ikke for mig, du må gerne lige teste på preview miljøet :D (Jeg kan ikke logge ind)